### PR TITLE
Make actions column header non-sortable

### PR DESCRIPTION
**Pull request by andreich1980**
Instead of my previous PR #3249 that I did wrong.

It fixes `.no-sort` classe on actions column header to prevent dataTable sorting by it.
Also fixes duplicate ID for `td`s in table rows.

### CODE REVIEW

File: resources/views/tools/database/index.blade.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
                     <thead>
                         <tr>
                             <th>{{ __('voyager::database.table_name') }}</th>
-                            <th style="text-align:right">{{ __('voyager::database.table_actions') }}</th>
+                            <th style="text-align:right no-sort">{{ __('voyager::database.table_actions') }}</th>
```
Comment (emptynick) [inline resources/views/tools/database/index.blade.php]: The database-tool doesn't use a Datatable, so none of your classes will kick in here

Comment (emptynick) [inline resources/views/tools/database/index.blade.php]: The database-tool doesn't use a Datatable, so none of your classes will kick in here

**Review by emptynick — CHANGES_REQUESTED** [review]

**Comment by emptynick** [comment]
As changing CSS is always breaking (needs republishing) why dont you simply use datatables `orderable` options?
Like
```
"columnDefs": [
    { "orderable": false, "targets": -1 }
]
```
See [here](https://datatables.net/reference/option/columns.orderable)

File: resources/views/tools/database/index.blade.php (hunk 1 above)
Comment (andreich1980) [reply resources/views/tools/database/index.blade.php]: oh, you're right. Thanks. What should I do now? Create one more commit that removes the class in this file?

Comment (emptynick) [reply resources/views/tools/database/index.blade.php]: Yes, in the end `resources/views/tools/database/index.blade.php` shouldn't be changed at all.

File: resources/views/tools/database/index.blade.php [hunk 2]
```diff
@@ -61,7 +61,7 @@ class="btn-sm btn-default">
                             </div>
                         </td>
 
-                        <td class="actions">
+                        <td class="actions bread-actions">
```
Comment (emptynick) [inline resources/views/tools/database/index.blade.php]: `.bread-actions` is also unneccessary as its a child of `#dataTable`

**Review by emptynick — CHANGES_REQUESTED** [review]

**Comment by andreich1980** [comment]
@emptynick  It's done.

**Comment by emptynick** [comment]
That looks much better.
Can you bring the `Actions` header to the right side in the menu-browse blade?

**Comment by andreich1980** [comment]
@emptynick I could add `.actions { text-aligh: right; }` but you said it's a breaking change. 
Or do you want me just add `style="text-align: right;"` on the actions column header?

**Review by emptynick — APPROVED** [review]

**Pull request merged**